analysis/generate_report.py

```diff
@@ -74,6 +76,11 @@ def get_arg_parser():
                         '--fuzzers',
                         nargs='*',
                         help='Names of the fuzzers to include in the report.')
+    parser.add_argument('-cov',
+                        '--coverage-report',
```

Comment:
consider something like `--link-coverage-reports` to indicate that they are not generated, just linked